index.js

```diff
@@ -1,15 +1,21 @@
 export const attention = {
   base: 'border-2 relative',
   tooltip:
-    'bg-gray-700 border-gray-700 text-white rounded-4 py-6 px-8',
-  callout: 'bg-green-100 border-green-400 py-8 px-16 rounded-8',
+    'i-bg-$color-tooltip-background i-shadow-$shadow-tooltip-shadow i-text-$color-tooltip-text rounded-4 py-6 px-8',
```

Comment:
Maybe something has changed but I don't think Fabric's Tooltip had any shadow 🤔 @adidick 
<img width="497" alt="Screenshot 2023-05-22 at 14 23 41" src="https://github.com/warp-ds/component-classes/assets/41303231/621bc52e-fad0-430f-b501-bab9e2e3d4e3">

Reply:
It did in Figma:
<img width="1108" alt="image" src="https://github.com/warp-ds/component-classes/assets/91149169/137ca850-43cc-4051-b04d-459443540ff2">

Reply:
It's a "medium" shadow

Reply:
But should this medium shadow be set as a token in case other brands want a different shadow or should we just set the medium shadow on the component level? I see @CarolineGard did the latter in [3a0c57e](https://github.com/warp-ds/component-classes/pull/57/commits/3a0c57e3f80a45d88336bdcc4daacc6d3c06a076)

Reply:
why should we not set the shadow on a component level, as we do for other components in the library? We should want the component to look and behave the same way everywhere?